packages/code-du-travail-frontend/src/outils/publicodes/Utils.ts

```diff
@@ -10,8 +10,19 @@ import { PublicodesResult, PublicodesUnit } from "./index";
 export const mapToPublicodesSituation = (
   form: FormContent
 ): Record<string, string> => {
-  const { ccn, infos, seniorityMaximum, seniorityValue, ...formWithoutCcn } =
-    form;
+  const {
+    ccn,
+    infos,
+    seniorityMaximum,
+    seniorityValue,
```

Comment:
je sais pas faire autrement pour exclure ces champs de `formWithoutCcn`

Si je le fait pas j'ai une erreur à cause des nouveau field que j'ai rajouté à `FormContent`
./src/outils/publicodes/Utils.ts:28:3
Type error: Type '{ "contrat salari\u00E9 - anciennet\u00E9": string; criteria?: Record<string, string> | undefined; cdt?: Record<string, string> | undefined; disabledWorker?: boolean | undefined; }' is not assignable to type 'Record<string, string>'.
  Property 'criteria' is incompatible with index signature.
    Type 'Record<string, string>' is not assignable to type 'string'.

Reply:
Je ne pense pas qu'on doit le traiter ici mais comme chaque simulateur possède ses propres données, il serait judicieux de typer le contenu du form selon le simulateur afin d'éviter ce genre de soucis. 
Je laisserai comme ça dans un premier temps du coup.

Reply:
Mais on aura  toujours le pb dans cette method qui est commune à tous les simulateurs non ?